modules/manage/partials/mountable-topics.adoc

```diff
@@ -0,0 +1,203 @@
+For topics with Tiered Storage enabled, you can mount and unmount topics to transfer the topic data between your cluster and object storage. This effectively frees up unused partition space that can be reclaimed, enables you to migrate a topic to a different cluster, and allows the topic or even the entire cluster to hibernate or be decommissioned.
+
+An unmounted topic in object storage is detached from all clusters. The original cluster releases ownership of the topic.
+
+You can mount a topic from object storage that has been previously flushed from a source cluster to a different destination cluster, or back to the same source cluster.
+
+The following data is moved to object storage when unmounting a topic from a cluster:
+
+* Topic definitions. 
+* The consumer offsets topic. This allows the destination cluster to also restore consumer group state.
+
+== Prerequisites
+
+. Enable xref:manage:tiered-storage.adoc[Tiered Storage] for specific topics, or for the entire cluster (all topics).
+. xref:get-started:rpk-install.adoc[Install `rpk`], or ensure that you have access to the Admin API.
+
```

Comment:
Are there any limitations that users should know about? For example, number of topics you can include in a migration? Amount of time that a topic can "hibernate" in object storage until it is mounted (although maybe that's more to do with their object storage configuration)?

Reply:
I'm not aware of any limitations like this. I've successfully ran it with as many topics and partitions as I had no problems to create.

Reply:
@bashtanov I saw in the [rpk PR](https://github.com/redpanda-data/redpanda/pull/23575/files#diff-560be64b94cb849f004c7d3849bb5336f8ff1990e63ec988e8732280a373d8efR38) that a topic must have at least 3 partitions for it to be mounted from TS to a cluster, does that apply anywhere in this doc as well?

Reply:
I never enforced or even heard about this restriction

Reply:
I saw that in the RFC for migrations, no idea if it is actually true